app/_src/using-mesh/managing-ingress-traffic/gateway-api.md

```diff
@@ -287,32 +423,34 @@ Gateway API isn't supported with multi-zone deployments, use {{site.mesh_product
 GAMMA is a Gateway API subproject focused on mesh implementations of Gateway API
 and extending the Gateway API resources to mesh use cases.
 
+{% if_version lte:2.6.x %}
 {% warning %}
-GAMMA itself as well as GAMMA in Kuma are **experimental**!
+GAMMA in {{site.mesh_product_name}} is **experimental**!
```

Comment:
```suggestion
GAMMA in {{site.mesh_product_name}} is **experimental**!
```

Is it? Maybe beta?

Reply:
It's wrapped in `{% if_version lte:2.6.x %}` block, so <= 2.6.x it's `experimental` >= 2.7.0 it's not

Reply:
Ah right, why change the message at all? Or at least maybe make it clear that at the time, GAMMA was also not yet GA?

Reply:
I guess in this case I would prefer that when looking at the docs for an older version I would see that the implementation, in that version, is for an experimental spec.

Reply:
But not a big deal